Assets/MixedRealityToolkit/Providers/UnityInput/GenericJoystickController.cs

```diff
@@ -118,24 +132,6 @@ protected void UpdateSingleAxisData(MixedRealityInteractionMapping interactionMa
 
             var singleAxisValue = UInput.GetAxis(interactionMapping.AxisCodeX);
 
-            switch (interactionMapping.InputType)
-            {
-                case DeviceInputType.TriggerPress:
```

Comment:
This change will also require some changes to the OpenVR WMR Controller, which also uses `SingleAxis` + `*Press` to simulate a digital input from an axis:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/89fb1f941856963b0012c6755ddaf61a94d4b999/Assets/MixedRealityToolkit.Providers/OpenVR/WindowsMixedRealityOpenVRMotionController.cs#L37

Either we update the grip axes there to be `Digital`, or we can reinstate this code and change the above `MixedRealityInteractionMapping`s to be `SingleAxis` as well.

Reply:
Interesting, that wasn't my experience but I'll test some more, thanks for catching this @keveleigh

Reply:
Addressed.   reinstated use case for testing TrigerPress used by WMR for Grip.

Ideally, this should be expressed as the two input types for Grip, position and press.  However, in analysis, this would have been a major breaking change that should be scoped later if required.

For now, the update reenables digital style input for the WMR trigger.